lib/web_of_science/harvester.rb

```diff
@@ -10,21 +10,30 @@ class Harvester < ::Harvester::Base
     # @param [Enumerable<Author>] authors
     # @return [void]
     def harvest(authors)
+      logger.info "#{self.class} - started harvest(authors) - #{authors.count} authors"
+      author_success = 0
       authors.each do |author|
-        begin
-          process_author(author)
-        rescue StandardError => e
-          NotificationManager.error(e, "WebOfScience harvest failed for author: '#{author.id}'", self)
-        end
+        process_author(author)
+        author_success += 1
       end
+      logger.info "#{self.class} - completed harvest(authors) - #{author_success} processed"
+    rescue StandardError => err
+      message = "#{self.class} - harvest(authors) failed"
```

Comment:
Can the rescue block here see the `author_success` variable?  Not entirely confident of the ruby scope rules on this one.  If it's in scope, sure, it could be added.

Reply:
Done and pushed.